contracts/ActiveBridgeSetLib.sol

```diff
@@ -28,10 +28,18 @@ library ActiveBridgeSetLib {
     uint256 lastBlockNumber;
   }
 
+  modifier validBlockNumber(uint256 _currentBlock, uint256 _lastBlock) {
```

Comment:
`current` and `last` are confusing to me in this context, as they could be seen as "currently in the contract state" or "last appended to the chain" (and both interpretations are wrong).

What about this?
```suggestion
  modifier validBlockNumber(uint256 _blockFromArguments, uint256 _blockFromContractState) {
```

Then the `require` instruction below would become super expressive as it would literally read _"require that block from arguments is greater or equal than block from contract state"_.